code/modules/modular_computers/computers/item/role_tablet_presets.dm

```diff
@@ -6,14 +6,20 @@
 	greyscale_config = /datum/greyscale_config/tablet/head
 	greyscale_colors = "#67A364#a92323"
 	max_capacity = parent_type::max_capacity * 2
-	starting_programs = list(
+	var/static/list/datum/computer_file/head_programs = list(
 		/datum/computer_file/program/crew_manifest,
 		/datum/computer_file/program/status,
 		/datum/computer_file/program/science,
 		/datum/computer_file/program/robocontrol,
 		/datum/computer_file/program/budgetorders,
 	)
 
+/obj/item/modular_computer/pda/heads/Initialize(mapload)
+	. = ..()
+	for(var/programs in head_programs)
+		var/datum/computer_file/program/program_type = new programs
+		store_file(program_type)
+
```

Comment:
Small thing, but this will put the "common" programs at the bottom of the installed list right?

Reply:
It looks that way based on the screen shot

Reply:
for record, I did try to do it without the initialize, and it just didn't load them at all, which is why I changed it to a static list and initialized it, which did work